react-apps/self-care-portal/self-care-portal-frontend/accelerator/src/specConfigs/common.js

```diff
@@ -17,61 +17,58 @@
  */
 
 export const common = {
-    footerContent: "WSO2 Open Banking | 2021",
-    complaintHandleLinkText: "Complaint handling and resolution",
+  footerContent: 'WSO2 Open Banking | 2024',
```

Comment:
Shall we retrieve the year from current date and add here

Reply:
Fixed this in [a80b0de](https://github.com/wso2/financial-open-banking/pull/49/commits/a80b0dee72da85a13bb6e2ca48216d1626320953)